Samples/CrossUnityDependencies.Unity/Assets/Dependencies.msb4u.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project ToolsVersion="15.0">
+  <!--GENERATED FILE-->
+  <!--
+    This file can be modified and checked in.
+    
+    It is different from the other generated C# Projects in that it will be the one gathering all dependencies and placing them into the Unity asset folder.
+    
+    You can add project level dependencies to this file, by placing them below:
+    - <Import Project="$(MSBuildForUnityGeneratedProjectDirectory)\$(MSBuildProjectName).g.props" />
+    and before:
+    - <Import Project="$(MSBuildForUnityGeneratedProjectDirectory)\$(MSBuildProjectName).g.targets" />
+    
+    Do not add any source or compliation items.
+    
+    Examples of how you can modify this file:
+    - Add NuGet package references:
+        <ItemGroup>
+          <PackageReference Include="Newtonsoft.Json" Version="11.0.1" />
+        </ItemGroup>
+    - Add external C# project references:
+      <ItemGroup>
+        <ProjectReference Include="..\..\..\ExternalLib\ExternalLib.csproj" />
+      </ItemGroup>
+  -->
```

Comment:
do you think we benefit from having this much content in all generated files? It kind of seems like if I was only shown a reference example this file would be less intimidating for someone new to csproj definitions.

Reply:
reducing the amount of content in this file will make it easier to read in things like notepad

---
In reply to: [367709527](https://github.com/microsoft/MSBuildForUnity/pull/108#discussion_r367709527) [](ancestors = 367709527)

Reply:
I will keep it for now, but will consider it in the future.

---
In reply to: [367709703](https://github.com/microsoft/MSBuildForUnity/pull/108#discussion_r367709703) [](ancestors = 367709703,367709527)